src/app/components/ModalExport.jsx

```diff
@@ -51,6 +55,10 @@ export default class ModalExport extends TranslatedComponent {
     }
   }
 
+  _copySLEF() {
+    navigator.clipboard.writeText(this.state.exportSLEF)
```

Comment:
This button won't be shown if the browser doesn't support the clipboard object. In this case the text will automatically be selected